packages/fields-markdown/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@keystone-alpha/fields-markdown",
+  "description": "KeystoneJS Markdown Field Type",
+  "version": "3.0.2",
```

Comment:
Should we reset this to `0.0.0` and let `changesets take care of things?

Reply:
Done (the version was only at 3.0.2 because I copied this package from somewhere else)